spec/services/idv/steps/verify_step_spec.rb

```diff
@@ -65,6 +65,20 @@
       step.call
     end
 
+    context 'when pii_from_doc is not present' do
+      let(:flow) do
+        Idv::Flows::DocAuthFlow.new(controller, {}, 'idv/doc_auth').tap do |flow|
+          flow.flow_session = { 'Idv::Steps::SsnStep' => true }
+        end
+      end
+
+      it 'does not 500' do
+        expect(flow.flow_session['Idv::Steps::SsnStep']).to eq true
+        step.call
```

Comment:
Maybe want to ensure that the `#call` result is empty, to protect in case a future maintainer decides the `nil` return in `mark_step_incomplete` is redundant?

Reply:
Good call, thank you, added